### PR TITLE
Features[1.memberDeletefromComunity 2.nameConvention in MemberDetails…

### PR DESCRIPTION
**Pull request by royanik17**
… 3. Remove CommunityId form member_list]

### CODE REVIEW

**Review by ayanit84 — CHANGES_REQUESTED** [review]
unit tests

File: src/main/java/com/experimental/product/community/membershipservice/client/request/CreateCommunityRequestV2.java [hunk 1]
```diff
@@ -32,6 +32,17 @@ public class CreateCommunityRequestV2 {
     @JsonProperty("has_pool")
     private Boolean hasPool;
 
+    public CreateCommunityRequestV2(String name, String description, String address, Map<ContactType, String> contacts, List<String> emergencyContacts, Boolean isGated, Boolean hasPool, Boolean hasGym) {
```
Comment (ayanit84) [inline src/main/java/com/experimental/product/community/membershipservice/client/request/CreateCommunityRequestV2.java:35]: please move the constructor below all member variables. 
1. first member variables
2. then constructors
3. then public methods

File: src/test/java/com/experimental/product/community/membershipservice/controller/CommunityAdminControllerV2Test.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+public class CommunityAdminControllerV2Test {
+
+    @Mock
+    private CommunityAdminServiceV2 communityAdminService;
+
```
Comment (ayanit84) [inline src/test/java/com/experimental/product/community/membershipservice/controller/CommunityAdminControllerV2Test.java:26]: Good Job

**Review by ayanit84 — APPROVED** [review]
2 comments

File: src/main/java/com/experimental/product/community/membershipservice/client/request/CreateCommunityRequestV2.java (hunk 1 above)
Comment (royanik17) [reply src/main/java/com/experimental/product/community/membershipservice/client/request/CreateCommunityRequestV2.java:35]: Ok sure

**Pull request merged**